examples/wat/spec/large_scheduled.wat

```diff
@@ -1,12 +1,13 @@
 (module
     (import "env" "memory" (memory 1))
     (import "env" "gr_reply_push" (func $reply_push (param i32 i32 i32)))
-    (import "env" "gr_reply_commit" (func $reply_commit (param i32 i32 i32)))
+    (import "env" "gr_reply_commit" (func $reply_commit (param i32 i32)))
     (export "init" (func $init))
     ;; ptr [0; 4) for gr_reply_push err length
     ;; ptr [4; 40) for gr_reply_commit err length
     ;; length of each push supposed to be a half of a wasm page (32 kb)
     ;; delay for sending will be 100
+    ;; TODO: add real delay for 100 blocks, see issue #2870
```

Comment:
I'd open new issue for that or implemented within the pr because all you need is just replace reply_push with send_push and gr_reply_commit with gr_send_commit with gr_source destination

Reply:
f87ab2e0a43321dff0faf64b1cb5984cf9d90cfe